.github/workflows/release.yml

```diff
@@ -1,31 +1,85 @@
-# Cut a release whenever a new tag appears on the repository.
-name: Release
+name: "Create release"
 
 on:
-  # Developers can manually push a tag from their clone.
-  push:
-    tags:
-      # NB: this assumes a consistent tagging scheme for the repo.
-      # If/when there are multiple Bazel modules, we may find that semver requires
-      # one to remain stable while another has breaking changes. In that case we'll
-      # have to have a tagging scheme that versions modules independently.
-      - "v*.*.*"
+  workflow_dispatch:
+    inputs:
+      version:
+        required: true
+        type: "string"
+
 permissions:
   id-token: write
   attestations: write
   contents: write
+
 jobs:
+  ci:
+    name: "CI"
+    uses: "./.github/workflows/ci.yaml"
+  
+  create_tag:
+    name: "Create tag"
+    runs-on:
+      - "ubuntu-latest"
+    needs:
+      - "ci"
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: "Create tag"
+        id: "tag"
+        env:
+          VERSION: "${{ inputs.version }}"
+        run: |
+          # 1. Download `buildozer`
+          curl -o "${RUNNER_TEMP}/buildozer" -L "https://github.com/bazelbuild/buildtools/releases/download/v7.3.1/buildozer-linux-amd64"
```

Comment:
I'd prefer if you used RUNNER_TOOL_CACHE instead and checked for the existence of this file first (maybe with some unconditional checksum validation).

Reply:
This runs on an ephemeral GH runner provided by GH. It won't be cached anyway.

This is what we're using in the public `engflowapis` repo. I think I'd prefer to merge it as is to test and then we can follow-up. WDYT

Reply:
Mind adding a TODO or note? This could be handy if we switched to some kind of hosted runners later

Reply:
`TODO(yannic)` is my favorite thing to leave everywhere ;)